src/renderers/BaseRenderer.php

```diff
@@ -157,6 +157,15 @@
      */
     public $extraButtons;
 
+    /**
+     * @var null|array CSS grid classes for horizontal layout. This must be an array with these keys:
+     *  - 'offset' the offset grid class to append to the wrapper if no label is rendered
+     *  - 'label' the label grid class
+     *  - 'wrapper' the wrapper grid class
+     *  - 'error' the error grid class
+     */
+    public $horizontalCssClasses = [];
```

Comment:
i don't like the name, may be just layoutConfig as array and use key `style` or `cssClasses` to manage css classes? 
Because there is no other layout except of horizontal for ListRenderer. Also we will be able use this property to extend functionality of layout  in the future  without introduce new property

Reply:
Then it will look like this:
```php
$widgetOptions = [
    'layoutConfig' => [
        'cssClasses' => [
            'offset' => 'col-sm-offset-4',    
            'label' => 'col-sm-4',
            'wrapper' => 'col-sm-4'
        ]
    ]
]
```
I think that in this case too many levels of nesting. And I do not know what to add for the property of the layoutConfig, except cssClasses

Reply:
it depends on you how to do it.

Reply:
i just don't like add new property for every minor feature. We can use one level of nesting and the following keys:
- offsetClass
- labeClass

and so on

what do you think?

Reply:
Okay. It will be much better.

Reply:
Ok then i'm waiting for this fix and also don't forget to update CHANGELOG and wiki 😄

Reply:
Done. Check, please)